server/service/src/sync/init_programs_data.rs

```diff
@@ -192,13 +192,30 @@ const VL_ELIGIBILITY_REPORT: &'static str =
 const LTFU_REPORT: &'static str = std::include_str!("./program_schemas/report_ltfu.json");
 const GBV_REPORT: &'static str = std::include_str!("./program_schemas/report_gbv.json");
 
+const EMPTY_CONTACT_DETAILS: ContactDetails = {
+    ContactDetails {
```

Comment:
Just checked you could also use `ContactDetails::default()`...

Reply:
Sorry, I'm not quite clear on the syntax I need for that. Can you be more specific?

Reply:
```rust
const EMPTY_CONTACT_DETAILS: ContactDetails = ContactDetails::default();
```
should work as well

Reply:
Right, that's what I tried and it didn't work. 🤷‍♂️
![Screenshot 2023-03-02 at 9 07 28 AM](https://user-images.githubusercontent.com/5456533/222253992-38ed17aa-243f-425f-ab08-41ad316f4114.png)

Reply:
Rust has a `Default` macro and if some types us it then you have this default() method... 

e.g.
```rust
#[derive(Default)]
struct MyStruct {
}
```

Reply:
Okay, I think I'll leave it for now. Feel free to tidy it up at your convenience. :)

Reply:
Oh yes sorry what I tried is to inline it:

vec![ContactDetails::default()]

but you can't call a method when initializing a const...

so leave as it is or inline it...